DATASETS.md

```diff
@@ -112,18 +112,39 @@ habitat-viewer --dataset '<PATH TO HM3D>/hm3d_annotated_basis.scene_dataset_conf
 
 To load semantic annotations in habitat-lab:
 
-* Add the semantic sensor to the list of agent sensors:
-
-    ```
-    SIMULATOR.AGENT_0.SENSORS = ["RGB_SENSOR", "SEMANTIC_SENSOR"]
+* Update your config by adding the semantic sensor to the list of agent sensors:
+
+    ```yaml
+    # @package _global_
+
+    defaults:
+      - /habitat: habitat_config_base
+      - /habitat/simulator/agents@habitat.simulator.agents.<agent_name>: agent_base
+      # add semantic sensor:
+      - /habitat/simulator/sim_sensors@habitat.simulator.agents.<agent_name>.sim_sensors.semantic_sensor: semantic_sensor
+      - /habitat/task/rearrange: <task>
+      - /habitat/dataset/rearrangement: <dataset>
+      - _self_
+    
+    habitat:
+      simulator:
+        agents:
+          <agent_name>:
```

Comment:
NIT: it's not quite clear that this is an variable when listed as key like this. Is there a sensible defualt we can provide them. Or at least add comments to those things which need to be changed.

Reply:
Thank you @Skylion007!
How about now?